XRTK-Core/Packages/manifest.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "com.unity.package-manager-ui": "2.1.2",
     "com.unity.textmeshpro": "2.0.1",
-    "com.unity.xr.legacyinputhelpers": "2.0.8",
+    "com.unity.xr.legacyinputhelpers": "2.1.3",
```

Comment:
We sure we want to do this?

Last time I updated this package we had some errors

Reply:
@StephenHodgson do you remember what happened? What the errors were, so I can verify.